requirements.txt

```
@@ -1,5 +1,6 @@
 click
 jinja2
+license_expression
```

This is both the name upstream uses and the PEP 503 normalized name:

```suggestion
license-expression
```